app/helpers/fhir_helper.rb

```diff
@@ -263,8 +265,8 @@ def language_coding(language)
   end
 
   # Helper to understand an extension for isolation status
-  def from_isolation_extension(patient)
-    patient&.extension&.select { |e| e.url.include?('isolation') }&.first&.valueBoolean == true
+  def from_bool_extension(patient, extension_id)
+    patient&.extension&.select { |e| e.url.include?(extension_id) }&.first&.valueBoolean == true
```

Comment:
Could this ever potentially mark an extension value improperly? What I mean is, lets say the payload didn't include the extension. `patient&.extension&.select { |e| e.url.include?(extension_id) }&.first&.valueBoolean` in that case would be null, and `nil == true` is false. I don't think false and nil are always the same thing.

Also, you can use `find` instead of `select` - and then you don't need the `.first`.

Reply:
Yes this is something we need to be careful of. This function is used on `isolation` and `continuous_exposure` as of right now, and both of those booleans have default values of `false`. So I think in the context it is being used, the absence of the extensions is equivalent to `false`. 

There is definitely risk that as we add future extensions we might blindly use this function without considering what the default value may be. I could rename the function so that it is clear it should only be used when the default is false? Or perhaps just add a strongly worded comment above the function. I'm leaning option 1, but let me know if you disagree.

Reply:
Or honestly I'm leaning both options 1 and 2.

Reply:
Yeah, 1 and 2 both sound perfect to me. I think the combinations of those would do the trick!

Reply:
Alright, I clarified the name and added a comment in https://github.com/SaraAlert/SaraAlert/pull/709/commits/44b6f15c8f53e62ce6c19ffe836d6e610707c808.